src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -649,6 +693,136 @@ public static String composePagedCallableMethodHeaderSampleCode(
     return bodyExprs.stream().map(e -> ExprStatement.withExpr(e)).collect(Collectors.toList());
   }
 
+  private static List<Statement> composeClientStreamMethodSampleCodeBodyStatements(
+      Method method,
+      VariableExpr clientVarExpr,
+      VariableExpr requestVarExpr,
+      AssignmentExpr requestAssignmentExpr) {
+    // Create responseObserver variable expression.
+    // e.g. ApiStream<EchoResponse> responseObserver
+    TypeNode responseObserverType =
+        TypeNode.withReference(
+            ConcreteReference.builder()
+                .setClazz(ApiStreamObserver.class)
+                .setGenerics(method.inputType().reference())
+                .build());
+    VariableExpr responseObserverVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("responseObserver").setType(responseObserverType).build());
+    // Create Anonymous Class contains methods: onNext, onError, onCompleted
+    // TODO (summerji): <pre> and {@code} is not supporting rendering '@' character, it is evaluated
+    // as Javadoc tag. Customize escape for '@' character.
+    MethodDefinition onNextMethod =
+        MethodDefinition.builder()
+            .setIsOverride(true)
+            .setScope(ScopeNode.PUBLIC)
+            .setName("onNext")
+            .setArguments(
+                VariableExpr.builder()
+                    .setVariable(
+                        Variable.builder().setName("response").setType(method.outputType()).build())
+                    .setIsDecl(true)
+                    .build())
+            .setBody(
+                Arrays.asList(
+                    CommentStatement.withComment(
+                        LineComment.withComment("Do something when receive a response."))))
```

Comment:
Let's fix this while we're here, e.g. "Do something when a response is received."